README.md

```diff
@@ -334,7 +337,7 @@ privateAggregation.enableDebugMode({debugKey: 1234n});
 ```
 
 This javascript function can only be called once per context. Any subsequent
```

Comment:
Maybe let's just delete this repeated paragraph, putting the ref under "Enabling" probably makes more sense :)

Reply:
Done!